src/Controls/src/Core/BindableLayout/BindableLayout.cs

```diff
@@ -256,6 +258,7 @@ static void SetBindableLayoutTemplate(BindableObject b, DataTemplate value)
 
 		public BindableLayoutController(IBindableLayout layout)
 		{
+			_dispatcher = (layout as BindableObject)?.Dispatcher;
```

Comment:
Instead of capturing the dispatcher here, would it be better to use the dispatcher from the layout when it is needed?

Reply:
@mattleibow good call
https://github.com/dotnet/maui/blob/9055699a90010990d24d3e7c77bf0d6204b6349b/src/Controls/src/Core/BindableObject.cs#L26-L27
I've just changed the code! Thanks